policy-controller/k8s/index/src/pod.rs

```diff
@@ -45,19 +45,19 @@ pub(crate) type PortMap<V> =
 #[derive(Debug, Default)]
 pub(crate) struct PortHasher(u16);
 
-/// Gets the set of named TCP ports from a pod spec.
-pub(crate) fn tcp_port_names(spec: Option<k8s::PodSpec>) -> HashMap<String, PortSet> {
+/// Gets the set of named non-UDP ports from a pod spec.
```

Comment:
Eh, I don't really like this definition. What if there are other protocol types (e.g., SCTP) in here? This is nitty, and I know it's all a bit confusing, but let's just be clearer about what this is actually doing.

```suggestion
/// Gets the set of named ports with `protocol: TCP` from a pod spec.
```